_html/scripts/build/script.js

```diff
@@ -266,9 +266,28 @@
             var sponsorCategory = $('.sponsors-categories--category');
 
             sponsorCategory.matchHeight();
+        },
+
+        burgerMenu: function() {
+            var menuButton = $('.burger-button');
+            var menuButtonLayers = $('.btn-layers');
+            var modalBurgerMenu = $('.burger-menu');
+            var burgermenuActionLink = $('.action-link');
+            var sublinks = $('.sublinks');
+        
+            menuButton.on('click', function() {
+                menuButtonLayers.toggleClass('btn-layers-fade');
+                modalBurgerMenu.fadeToggle();
+                
+                burgermenuActionLink.on('click', function() {
```

Comment:
Why is this event listener nested inside the menuButton listener?

Reply:
its secondary to previcious listener, conditioned by him.

Reply:
But why do you need it to depend on the other one? Why not just simply add it?
It doesn't matter if the element is visible or not. If it's added to the DOM, you can add the listener.